index.php

```diff
@@ -13,11 +13,57 @@
  * @since   Timber 0.1
  */
 
+use P4\MasterTheme\Features\Dev\ListingPageGridView;
+use P4\MasterTheme\Features\ListingPagePagination;
+use Timber\Timber;
+
 $context          = Timber::get_context();
 $context['posts'] = Timber::get_posts();
+$context['title'] = html_entity_decode( $context['wp_title'] );
 
 $templates = [ 'index.twig' ];
+
 if ( is_home() ) {
-	array_unshift( $templates, 'home.twig' );
+	array_unshift( $templates, 'all-posts.twig' );
+
+	if ( ListingPagePagination::is_active() ) {
+		$view = ListingPageGridView::is_active() ? 'grid' : 'list';
+
+		$query_template = file_get_contents( get_template_directory() . "/parts/query-$view.html" );
+
+		$content = do_blocks( $query_template );
+
+		$context['query_loop'] = $content;
+		Timber::render( $templates, $context );
+		exit();
+	} else {
+		// Only applied to the "Load More" feature.
+		if ( null !== get_query_var( 'page_num' ) ) {
+			$wp_query->query_vars['page'] = get_query_var( 'page_num' );
+		}
+	}
+
+	$post_args = [
+		'posts_per_page' => 10,
+		'post_type'      => 'post',
+		'paged'          => 1,
+		'has_password'   => false,  // Skip password protected content.
+	];
+
+	if ( get_query_var( 'page' ) ) {
```

Comment:
I prefer to keep the same logic as the other listing pages and only adding [this logic](https://github.com/greenpeace/planet4-master-theme/pull/1773/files#diff-7413d6453f901e939bbd840c8f0d1c7b20c2ca0e7f71741e4e07c6cf036f16c0R40-R43)